src/features/user_profile/components/SettingsForm/ReferenceAreaInfo/ReferenceAreaInfo.tsx

```
@@ -43,11 +93,32 @@ export function ReferenceAreaInfo() {
+              <span className={s.userLocationLoader}>
+                {i18n.t('profile.reference_area.accessing_location')}
+              </span>
+            ) : locationLoadError ? (
+              <span className={s.errorMessage}>
+                {i18n.t('profile.reference_area.accessing_location_error')}
+              </span>
+            ) : (
+              <button
+                className={clsx(s.link, s.userLocation)}
+                onClick={onSelectCurrentLocation}
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)